api-definitions/db_positions/api.yaml

```diff
@@ -0,0 +1,423 @@
+    minLength: 1
+    example: ufficio tributi comune di Roma
+  Taxonomy:
+    type: string
+    pattern: "^[0-9]{3}-[a-zA-Z]$"
+    example: 010-TARI/TEFA
+    description: Payment Tax see [Tabella - Tassonomia dei servizi di incasso](https://docs.google.com/spreadsheets/d/13xOd__Qd4pwKHr3wjE-73NAB2O7UKmIt/edit#gid=1444196601)
+  Transfer:
+    type: object
+    properties:
+      partial_amount:
+        $ref: "#/definitions/Amount"
+      iban:
+        $ref: "#/definitions/Iban"
+      organization_fiscal_code:
+        $ref: "#/definitions/OrganizationFiscalCode"
+      reason:
+          type: string
+          minLength: 1
+          example: Causale pagamento TARI comune di ROMA 
+      taxonomy:
+          $ref: "#/definitions/Taxonomy"
+    required:
+      - partial_amount
+      - iban
+      - organization_fiscal_code
+      - taxonomy
+  Psp:
+    type: object
+    properties:
+      id:
+        type: string
+        minLength: 1
+        example: psp1234567890
+      company_name:
+        type: string
+        minLength: 1
+        example: PSP company name
+      id_channel:
+        type: string
+        minLength: 1
+        example: PSP'channel identification
+      description:
+        type: string
+        minLength: 1
+        example: PSP'description
+  Debtor:
+    type: object
+    properties:
+      name:
+        type: string
+        minLength: 1
+        example: Mario
+      surname:
+        type: string
+        minLength: 1
+        example: Rossi
+      fiscal_code:
+        $ref: "#/definitions/FiscalCode"
+      type:
+        type: string
+        enum:
+          - FISICA
+          - GIURIDICA
+        description: Payment Status  
+    required:
+      - name
+      - surname
+      - type
+  Payment-Option:
+    type: object
+    properties:
+      id:
+        $ref: "#/definitions/PaymentOptionId"
+      amount:
+        $ref: "#/definitions/Amount"
+      due_date:
+        $ref: "#/definitions/Timestamp"
+      is_conclusive:
+        type: boolean
+        default: true
+      payment_description: 
+        type: string
+        minLength: 1
+        example: TARI 2021 soluzione unica
+      transfers:
+        type: array
+        items:
+          $ref: "#/definitions/Transfer"
+    required:
+      - id
+      - amount
+      - due_date
+      - is_conclusive
+      - payment_description
+      - transfers
+  ReceiptPayment:
+    type: object
+    properties:
+      id:
+        type: string
+        minLength: 1
+        example: 123e4567-e89b-12d3-a456-426614174000
+        description: Receipt identification ( generated by NodoSPC )
+      notice_id:
+        $ref: "#/definitions/PaymentNoticeNumber"
+      option_id:
+        $ref: "#/definitions/PaymentOptionId"
+      organization_fiscal_code:
+        $ref: "#/definitions/OrganizationFiscalCode"
+      outcome:
+        type: string
+        enum:
+          - OK
+          - NOK
+      # Organization data
```

Comment:
When you have a flat hierarchy then using a prefix is a good idea to group related fields (ie. `organization_`). 
Another option is to add some depth using inner objects (ie. `organization_data: { debtor: ..., psp: ... }`) so you may have a schema reference ie. `organization_data: $ref: "#/definitions/OrganizationData"`) which make the spec self documenting